static/sass/_patterns_footer.scss

```diff
@@ -0,0 +1,234 @@
+    padding-top: 0;
+
+    @media only screen and (min-width: $breakpoint-medium) {
+      padding-top: .75rem;
+    }
+
+    @media only screen and (max-width: $breakpoint-medium) {
+      &__nav-col {
+        width: 100%;
+      }
+    }
+
+    &__nav {
+      border-bottom: 1px solid $color-mid-light;
+
+      @media only screen and (min-width: $breakpoint-medium) {
+        border: 0;
+        padding-top: 40px;
+      }
+    }
+
+    &__divider {
+      @include full-width-breakout;
+      border: 0;
+      border-bottom: 1px dotted $color-x-dark;
+      border-collapse: separate;
+    }
+
+    &__links {
+      margin: 0;
+      padding: 0;
+    }
+
+    &__item {
+      display: block;
+      margin-bottom: 0;
+
+      @media only screen and (min-width: $breakpoint-medium + 1) {
+        margin-bottom: 1.5rem;
+      }
+    }
+
+    &__title {
+      @include footer-title;
+      background-image: url("#{$assets-path}7bd1bd7b-arrow_down_9fa097.svg");
+      background-position: calc(100% - 10px) 50%;
+      background-repeat: no-repeat;
+      background-size: 13px 13px;
+      border-top: 1px solid $color-mid-light;
+      font-size: .87rem;
+      padding: 10px 0;
+
+      @media only screen and (min-width: $breakpoint-medium) {
+        background: none;
+        border: 0;
+        color: $color-dark;
+        padding: 0 0 6px;
+      }
+
+      @media only screen and (min-width: $breakpoint-large) {
+        font-size: .8125rem;
+      }
+
+      a:link,
+      a:visited {
+        color: $color-mid-dark;
+        padding-left: 10px;
+
+        @media only screen and (min-width: $breakpoint-medium) {
+          color: $color-dark;
+          font-weight: normal;
+          padding-left: 0;
+        }
+      }
+
+      a::after {
+        content: ''; // hide the default > */
+      }
+
+      /// open and close the primary/secondary navs
+      + .second-level-nav {
+        display: none;
+
+        @media only screen and (min-width: $breakpoint-medium) {
+          display: block;
+        }
+      }
+
+      &.active {
+        background-image: url("#{$assets-path}43e2b367-arrow_up_9fa097.png");
+
+        @media only screen and (min-width: $breakpoint-medium) {
+          background: none;
+        }
+
+        + .second-level-nav {
+          display: block;
+        }
+      }
+    }
+
+    &__container {
+      @media only screen and (max-width: $breakpoint-medium - 1px) {
+        padding-left: 0;
+        padding-right: 0;
+      }
+    }
+
+    &__link {
+      &:hover {
+        color: $color-dark;
+      }
+    }
+
+    .link-to-top {
+      @include footer-title;
+      background-image: url("#{$assets-path}e4a5a8c0-picto-upload-warmgrey.svg");
+      background-position: 10px center;
+      background-repeat: no-repeat;
+      background-size: 14px 14px;
+      border-bottom: 1px solid $color-mid-dark;
+      display: block;
+      left: auto;
+      margin-bottom: -1px;
+      padding: 10px 0 10px 30px;
```

Comment:
I would recommend not rems but Vanilla spacing variable - in cases where there is not 1:1 match but they are only a couple of pixels out and doesn't break anything, use the spacing variable. Remember we have agreed with design that this migration will not be a pixel perfect representation, the whole point of using a framework is to let the framework do the heavy lifting.